Tests/AppTests/Controllers/Event/EventSpeechControllerTests.swift

```diff
@@ -45,8 +45,8 @@ class EventSpeechControllerTests: TestCase {
     
     try storeSpeech(for: eventId)
 
-    let res = try! fetchSpeeches(by: id)
-    let speechJSON = res.json?.array?.first
+    let result = try! fetchSpeeches(by: id)
```

Comment:
I think `res` here was for `response` not `result`.

Reply:
sure, renamed!